deltachat-ios/Chat/Send Reaction/SendReactionsView.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+import DcCore
+
+enum DefaultReactions: CaseIterable {
+    case thumbsUp
+    case thumbsDown
+    case heart
+    case haha
+    case sad
+
+    var emoji: String {
+        switch self {
+        case .thumbsUp: return "👍"
+        case .thumbsDown: return "👎"
+        case .heart: return "❤️"
+        case .haha: return "😀"
```

Comment:
i've just noticed, we're using 😂 on android/ios; we should use the same here then

```suggestion
        case .haha: return "😂"
```